### PR TITLE
Added type hints to additional tests

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: Tests/test_image_thumbnail.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 def test_sanity() -> None:
     im = hopper()
-    assert im.thumbnail((100, 100)) is None
+    assert im.thumbnail((100, 100)) is None  # type: ignore[func-returns-value]
```
Comment (radarhere) [inline Tests/test_image_thumbnail.py:19]: Without this ignore, mypy reports

> Tests/test_image_thumbnail.py:19: error: "thumbnail" of "Image" does not return a value (it only ever returns None)  [func-returns-value]
>         assert im.thumbnail((100, 100)) is None
>                ^~~~~~~~~~~~~~~~~~~~~~~~

Comment (Yay295) [reply Tests/test_image_thumbnail.py:19]: Couldn't this line just be `im.thumbnail((100, 100))`, without the assert?

Comment (radarhere) [inline Tests/test_image_thumbnail.py:19]:
```suggestion
    im.thumbnail((100, 100))
```

Comment (radarhere) [reply Tests/test_image_thumbnail.py]: Ok, I’ve pushed a commit.

**Pull request merged**